network-api/networkapi/news/models.py

```diff
@@ -88,7 +89,7 @@ class News(models.Model):
 
     objects = NewsQuerySet.as_manager()
 
-    class Meta:
+    class Meta(TranslatableMixin.Meta):
```

Comment:
Let's check if this is actually necessary or not. If we don't need to mark the Meta class as translatable, that saves some migration work. (not much, but "not doing anything we don't need to do is worth not doing" =D)